src/Symfony/Component/Notifier/Bridge/Infobip/Tests/InfobipTransportTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Infobip\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Notifier\Bridge\Infobip\InfobipTransport;
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+final class InfobipTransportTest extends TestCase
+{
+    public function testToStringContainsProperties(): void
+    {
+        $transport = $this->initTransport();
+
+        $this->assertSame('infobip://host.test?from=0611223344', (string) $transport);
+    }
+
+    public function testSupportsMessageInterface(): void
+    {
+        $transport = $this->initTransport();
+
+        $this->assertTrue($transport->supports(new SmsMessage('0611223344', 'Hello!')));
+        $this->assertFalse($transport->supports($this->createMock(MessageInterface::class), 'Hello!'));
+    }
+
+    public function testSendNonSmsMessageThrowsException(): void
+    {
+        $transport = $this->initTransport();
+
+        $this->expectException(LogicException::class);
+
+        $transport->send($this->createMock(MessageInterface::class));
+    }
+
+    private function initTransport(): InfobipTransport
```

Comment:
I would call it `getTransport()` (I've renamed it in Freemobile tests).

Reply:
Fixed!